src/main/webapp/app/exercises/shared/result/result.utils.ts

```diff
@@ -315,9 +315,9 @@ export function getTestCaseNamesFromResults(results: ResultWithPointsPerGradingC
         if (!result.result.feedbacks) {
             return [];
         }
-        result.result.feedbacks.map((f) => {
+        result.result.feedbacks.forEach((f) => {
             if (Feedback.isTestCaseFeedback(f)) {
```

Comment:
Could use `filter(feedback => feedback.isTestCaseFeedback(f)` and reduce nesting